### DISCUSSION TITLE
Low fuel

### DISCUSSION THREAD

**Discussion by mickmcp** [flybywiresim/aircraft#7576] [General]
All of a sudden when the aircraft loads onto the runway it is nearly empty of fuel. This has never happened before and usually has adequate fuel for most of the flights I do.
Its not a problem as I can re-fuel from the fly pad but I was just curious why its started doing this .

**Comment by ExampleWasTaken** [comment]
Given you are on the latest version of the a32nx, it will store your FOB from your last flight. So if you're closing the sim with only 3 tons FOB that value will be used the next time you load into the sim.
